elasticdl/python/ps/parameters.py

```diff
@@ -197,3 +197,24 @@ def to_model_pb(self):
             model_pb.embedding_table_info.append(embedding_info)
 
         return model_pb
+
+    def debug_info(self):
```

Comment:
Add something like `self._debug_info_needed`, and set it as True only when log_level=DEBUG. `debug_info` can return directly when `self._debug_info_needed==False` and no need to iterate over all variables.

Reply:
Currently, `debug_info` is only used in only one place, inside `parameter_server.run` method.

```python
self.logger.debug(
                    "Parameters info:\n%s" % self.parameters.debug_info()
                )
```

This line is only valid when log_level=DEBUG. I think it's no need to add such flag `self._debug_info_needed`.

Reply:
`self.parameters.debug_info()` will be executed even if `log_level==INFO`. Only the logger is not shown in the terminal.

Reply:
Done